install/ui/src/freeipa/host.js

```diff
@@ -130,6 +130,18 @@ return {
                                 {
                                     label: 'radius',
                                     value: 'radius'
+                                },
+                                {
+                                    label: 'pkinit',
+                                    value: 'pkinit'
+                                },
+                                {
+                                    label: 'spake',
+                                    value: 'spake'
+                                },
+                                {
+                                    label: 'fast',
+                                    value: 'fast'
```

Comment:
And here all the labels should be changed to the proper ones as `@i18n:authtype.type_<method>` for each method, even for the original ones.